src/amo/components/CollectionManager/index.js

```diff
@@ -317,11 +345,21 @@ export class CollectionManagerBase extends React.Component<Props, State> {
             value={this.state.slug}
           />
         </div>
-        {this.state.addonAddedStatus === ADDON_ADDED_STATUS_SUCCESS && (
-          <Notice type="success">
-            {i18n.gettext('Added to collection')}
-          </Notice>
-        )}
+
+        <ReactCSSTransitionGroup
+          className="NoticePlaceholder"
+          component="div"
+          transitionName="overlay"
```

Comment:
Is it a predefined name or custom?

Reply:
No i meant to change that... thanks for catching